services/apps/git_integration/src/crowdgit/services/clone/clone_service.py

```diff
@@ -186,13 +230,20 @@ async def _calculate_batch_depth(self, repo_path: str, remote: str) -> int:
 
     async def clone_batches(
```

Comment:
NIT: would it be helpful if we change the name to `clone_batches_generator` to show this is a function that yields results